tests/sftpsync_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+
+import os.path as op
+class MyTestCase(unittest.TestCase):
+    def test_connection(self):
+        from sftpsync import Sftp
+
+        sftp = Sftp('147.228.47.162', 'paul', 'P4ul')
```

Comment:
Hi,
just some quick thoughts on that commit:
I wouldn't post my server access if I were you.
Besides, should you ever change that access the test would break. A cleaner way to do it would be to set up a local sftp-server with paramiko and generate some dummy files to sync.

Best regards
Sebastian